Taskfile.yml

```diff
@@ -108,6 +108,7 @@ tasks:
         terraform init \
         -backend-config="bucket=${ARTIFACTS_BUCKET_NAME}" \
         -backend-config="prefix=${ARTIFACTS_BUCKET_TERRAFORM_PREFIX}"
+      - terraform validate -no-color
```

Comment:
Noice 👍 

`-no-color - Disables terminal formatting sequences in the output. Use this if you are running Terraform in a context where its output will be rendered by a system that cannot interpret terminal formatting.`